libs/components/src/lib/text-area/text-area.scss

```
@@ -26,9 +29,9 @@ $low-ink-color: --_low-ink-color;
 	@include connotation.connotation;
 
 	display: inline-grid;
-	width: 100%;
-	gap: 4px;
+	width: inherit;
 	grid-template-columns: 1fr max-content;
```

```suggestion
	grid-template-columns: 1fr;
```
not relevant to this. PT but there are no case where there are 2 columns in the grid